mitodl/open-discussions
# Update indexing methods to reduce data sent to and from redis via celery

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2518 

#### What's this PR do?
Converts create and update indexing methods to take a primary key value and pass it to the task to index, instead of serializing the data then sending it to the task. This should cut down on the amount of data in each task redis is keeping track of. 

#### How should this be manually tested?
- For videos, run youtube ETL task (backpopulate youtube data)
 - User list: create a user list and verify that 
 - Courses/programs: xpro integration (backpopulate xpro data)
